doc-content/drools-docs/src/main/asciidoc/DecisionEngine/phreak-sequential-mode-con.adoc

```diff
@@ -0,0 +1,83 @@
+[id='phreak-sequential-mode-con_{context}']
+
+= Sequential mode in Phreak
+
+Sequential mode is an advanced rule base configuration in the {DECISION_ENGINE}, supported by Phreak, that enables the {DECISION_ENGINE} to evaluate rules one time in the order that they are listed in the {DECISION_ENGINE} agenda without regard to changes in the working memory. In sequential mode, the {DECISION_ENGINE} ignores any `insert`, `modify`, or `update` statements in rules and executes rules in a single sequence. As a result, rule execution may be faster in sequential mode, but important updates may not be applied to your rules.
+
+Sequential mode applies to only stateless KIE sessions because stateful KIE sessions inherently use data from previously invoked KIE sessions. If you use a stateless KIE session and you want the execution of rules to influence subsequent rules in the agenda, then do not enable sequential mode. Sequential mode is disabled by default in the {DECISION_ENGINE}.
+
+To enable sequential mode, use one of the following options:
+
+* Set the system property `drools.sequential` to `true`.
+* Enable sequential mode while creating the KIE base in the client code:
++
+[source,java]
+----
+KieServices services = KieServices.Factory.get();
```

Comment:
I worry about making this change. `KieServices.Factory.get( )` is used everywhere throughout our docs project, including very new code supplied by other SMEs, like for PMML and DMN invocation from this 7.4 release even. In our entire docs project, there are currently only 2 instances of `KieServcies.get`, but 76 instances of `KieServices.Factory.get`. I can do a mass update, but we need to be absolutely certain because it will affect does of all walks (rules, processes, DMN, PMML, packaging and deployment, etc.).

Reply:
In this case leave it as it is. Both options are valid. KieServices.get() is just the shorter one.

Reply:
Okay, leaving.